src/index.ts

```diff
@@ -14,18 +15,21 @@ import { verifier, verify } from "rx-nostr-crypto";
 import { firstValueFrom, lastValueFrom } from "rxjs";
 
 const app = new Hono();
-const cache = caches.default;
 const eoseTimeout = 3000;
 const okTimeout = 3000;
 
+const cacheOptions = {
+  cacheName: "restr-v0",
```

Comment:
バージョン毎に変更はしないと思うのでバージョン情報はなしで。

```suggestion
  cacheName: "restr",
```